platform/core.windows/src/org/netbeans/core/windows/WindowManagerImpl.java

```diff
@@ -527,6 +579,11 @@ public void removePropertyChangeListener (PropertyChangeListener list) {
         public void setBounds (Rectangle s) {
             wrap.setBounds (s);
         }
+
+        @Override
+        public String toXml() {
```

Comment:
The fact that you have to add this method signals you made an incompatible change in `Mode`.